### PR TITLE
Detail code review process

### PR DESCRIPTION
**Pull request by tlestang**
This adds the content of the page describing the structure of the code review meetings themselves. The content was collaboratively edited over the past few meetings, and is accessible on [HackMD](https://hackmd.io/@researchcodereview/H1kE3pVcK). Last edit: 17.02.2022 meeting.

### CODE REVIEW

**Comment by bielsnohr** [comment]
Given I was involved in crafting most of this, I don't have much to add. There are two practical points I want to raise:

1. We should probably change the name of the file where we have made the outline. `review_meetings.md` is only a sub-topic of this section. Something like `perform_code_review.md`.
2. This raises the question about where this content will eventually sit and how we are going to collaboratively add content. Are we going to overwrite this outline with the content? Or are we going to make a separate file? Also, if we are going to tackle "Perform Code Review" in one file, with multiple people editing concurrently, we will likely encounter some merge conflicts. This perhaps suggests separate documents for each subsection of "Perform Code Review", but then I worry about how many links we are getting visitors of the site to click through.
    - My 2 cents: probably fine to just "overwrite" this outline with the actual content and worry about whether to create separate files later.

File: site/content/recipes/review_meetings.md [hunk 1]
```diff
@@ -1,8 +1,85 @@
 # Perform code review
 Draft outline base on  [flowcharts/high-level.md](https://github.com/ResearchCodeReviewCommunity/dev-review/blob/main/site/content/flowcharts/high-level.md#perform-code-review)
```
Comment (NickleDave) [inline site/content/recipes/review_meetings.md:2]: We can remove this, right?

File: site/content/recipes/review_meetings.md [hunk 2]
```diff
@@ -1,8 +1,85 @@
...
+
+This is [recipes/review_meetings.md](https://github.com/ResearchCodeReviewCommunity/dev-review/blob/main/site/content/recipes/review_meetings.md). You can edit the file normally (locally) and push to the GitHub remote to collaborate.
```
Comment (NickleDave) [inline site/content/recipes/review_meetings.md:4]: same here: this is text from HackMD that we can remove? Or is there something I'm not understanding?

**Comment by NickleDave** [comment]
Agree with @bielsnohr, :+1: for changing the document name to `perform-code-review.md`

File: site/content/recipes/review_meetings.md [hunk 3]
```diff
@@ -1,8 +1,85 @@
...
+
+You can also work directly in the Markdown web-editor (on the left hand side), 
+and save a version of this markdown file once you're happy with the changes you made (menu -> Versions and GitHub sync). We can also push this document to the GitHub remote from the same menu. Otherwise this works like a Google document.
+
 ## Code author communicates code and context
-## Review reviews code
```
Comment (NickleDave) [inline site/content/recipes/review_meetings.md:4]: I'm confused by this heading, `## Review reviews code` -- should it be here?

**Review by NickleDave — CHANGES_REQUESTED** [review]
LGTM plus or minus a couple minor edits, we can discuss at the next meeting

File: site/content/recipes/review_meetings.md (hunk 3 above)
Comment (tlestang) [reply site/content/recipes/review_meetings.md:4]: Well this line was removed no?

Comment (NickleDave) [reply site/content/recipes/review_meetings.md:4]: Ah of course, sorry, nvm

**Review by NickleDave — APPROVED** [review]

**Pull request merged**